src/CodegenClass.php

```diff
@@ -33,56 +35,76 @@
   private bool $isAbstract = false;
   private ?CodegenConstructor $constructor = null;
 
+  /** @selfdocumenting */
   public function setIsFinal(bool $value = true): this {
     $this->isFinal = $value;
     return $this;
   }
 
+  /** @selfdocumenting */
   public function setIsAbstract(bool $value = true): this {
     $this->isAbstract = $value;
     return $this;
   }
 
+  /** Set the parent class of the generated class. */
```

Comment:
Interesting that this wasn't called `setParent`

Reply:
I assumed appendBodyToBuilder was from an interface, but it isn't

I'm only saying 'parent' here to not re-use 'extends' - it's extends to match the keyword in generated code.